src/ui/Tab/Tab.ts

```diff
@@ -254,6 +254,9 @@ export class Tab extends Component {
     this.bind.on(element, 'click', clickAction);
     this.bind.on(element, 'keyup', KeyboardUtils.keypressAction(KEYBOARD.ENTER, clickAction));
     this.render();
+    if (this.options.layout) {
+      this.queryStateModel.set(QUERY_STATE_ATTRIBUTES.LAYOUT, this.options.layout);
```

Comment:
Wouldn't this set the layout _for each tab that has data-layout in the page_ and keep the last one executed?

It is possible to add a test to validate that with two tabs with different layouts, the first selected one should have its layout set?